nps-unit-sample.json

```diff
@@ -1,367 +1,367 @@
-          "rationaleConfidence": "The degree of confidence at Acadia NP is is high because there is an on-site or nearby visibility monitor."
-        },
-        {
-          "year": 2001,
-          "conditionStatus": "Warrants Moderate Concern",
-          "conditionTrend": "Relatively Unchanged",
-          "conditionConfidence": "High Confidence",
-          "conditionSummaryCombined": "concern-unchanging-high",
-          "benchmarkChartYAxisLabel": "Haze Index, dv",
-          "benchmarkChartValue": 11.5,
-          "benchmarkChartConcern": 15,
-          "benchmarkChartAverage": 12,
-          "benchmarkChartGood": 9,
-          "benchmarkChartColor": "Yellow",
-          "rationaleCondition": "Visibility warrants moderate concern at Acadia NP. This status is based on <a href='https://www.nps.gov/subjects/air/analysis-methods.htm'>NPS Air Resources Division benchmarks</a> and the 2011–2015 estimated visibility on mid-range days of 11.6 deciviews (dv), which is 4.4 dv above the estimated natural condition of 7.2 dv.",
-          "rationaleTrend": "For 2006–2015, the trend in visibility at Acadia NP improved on the 20% clearest and 20% haziest days, resulting in an overall improving visibility trend (IMPROVE Monitor ID: ACAD1, ME). The Clean Air Act visibility goal requires visibility improvement on the 20% haziest days, with no degradation on the 20% clearest days.",
-          "rationaleConfidence": "The degree of confidence at Acadia NP is is high because there is an on-site or nearby visibility monitor."
-        },
-        {
-          "year": 2002,
-          "conditionStatus": "Warrants Moderate Concern",
-          "conditionTrend": "Relatively Unchanged",
-          "conditionConfidence": "High Confidence",
-          "conditionSummaryCombined": "concern-unchanging-high",
-          "benchmarkChartYAxisLabel": "Haze Index, dv",
-          "benchmarkChartValue": 11.5,
-          "benchmarkChartConcern": 15,
-          "benchmarkChartAverage": 12,
-          "benchmarkChartGood": 9,
-          "benchmarkChartColor": "Yellow",
-          "rationaleCondition": "Visibility warrants moderate concern at Acadia NP. This status is based on <a href='https://www.nps.gov/subjects/air/analysis-methods.htm'>NPS Air Resources Division benchmarks</a> and the 2011–2015 estimated visibility on mid-range days of 11.6 deciviews (dv), which is 4.4 dv above the estimated natural condition of 7.2 dv.",
-          "rationaleTrend": "For 2006–2015, the trend in visibility at Acadia NP improved on the 20% clearest and 20% haziest days, resulting in an overall improving visibility trend (IMPROVE Monitor ID: ACAD1, ME). The Clean Air Act visibility goal requires visibility improvement on the 20% haziest days, with no degradation on the 20% clearest days.",
-          "rationaleConfidence": "The degree of confidence at Acadia NP is is high because there is an on-site or nearby visibility monitor."
-        }
-      ],
-      "monitoringSites": [
-        {
```

Comment:
monitoring sites in an array, each with their own charts and year-data.

Reply:
Okay.